src/main/java/org/apache/sling/maven/feature/launcher/StartMojo.java

```diff
@@ -93,6 +93,10 @@ public class StartMojo extends AbstractMojo {
      *   </framweworkProperties>
      *   ..
      * </launcherArguments>
+     * <repositoryUrls>
```

Comment:
can you add a comment what repositories are used for in general and also what the default is (Maven Central I guess).

Reply:
@kwin  - see https://github.com/apache/sling-feature-launcher-maven-plugin/pull/15/commits/158ca31f7ac517ac77233d6292553fa1b0bd29be